### ISSUE TITLE
UltiSnips generation fails

### ISSUE THREAD

**Issue by chauncey-garrett** [pearofducks/ansible-vim#58]
I get this when installing the snippets:

```
Traceback (most recent call last):
  File "generate.py", line 99, in <module>
    for document in get_documents():
  File "generate.py", line 17, in get_documents
    documentation = plugin_docs.get_docstring(os.path.join(root, f))[0]
TypeError: get_docstring() takes at least 2 arguments (1 given)
```

`ansible` is v2.5.

**Comment by pearofducks** [comment]
Can you try the `fix-generate-against-2.5` branch?

**Comment by cyberang3l** [comment]
I can confirm the fix in branch `fix-generate-against-2.5` works for ansible v2.5.

**Comment by pearofducks** [comment]
Thanks @cyberang3l  -  much appreciated!
